pkgs/by-name/qt/qt-scrcpy/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libsForQt5,
+  scrcpy,
+}:
+stdenv.mkDerivation rec {
+  pname = "qt-scrcpy";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "barry-ran";
+    repo = "QtScrcpy";
+    rev = "v${version}";
+    hash = "sha256-PL/UvRNqvLaFuvSHbkJsaJ2nqRp5+ERM+rmlKVtbShk=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = with libsForQt5; [
+    qtbase
+    wrapQtAppsHook
+    qtmultimedia
+    scrcpy
+    qtx11extras
+  ];
```

```suggestion
  nativeBuildInputs = [
    cmake
    libsForQt5.wrapQtAppsHook
  ];

  buildInputs = [ scrcpy ] ++ (with libsForQt5; [
    qtbase
    qtmultimedia
    qtx11extras
  ]);
```

(Please apply formatting)